salt/engines/slack.py

```diff
@@ -231,7 +231,7 @@ def get_config_groups(self, groups_conf, groups_pillar_name):
                 ret_groups[name]['default_target'].update(config.get('default_target', {}))
                 ret_groups[name]['targets'].update(config.get('targets', {}))
             except (IndexError, AttributeError):
-                log.warn("Couldn't use group %s. Check that targets is a dict and not a list", name)
+                log.warning("Couldn't use group %s. Check that targets is a dict and not a list", name)
```

Comment:
Messages would be also expand to a end-user level language. "Dict" ? A dictionary or key/value structure etc.